systemtest/src/test/java/io/strimzi/systemtest/StrimziUpgradeST.java

```diff
@@ -48,8 +46,18 @@ private void copyModifyApply(File root) {
         });
     }
 
+    private void deleteInstallYamls(File root) {
```

Comment:
should it be more `deleteInstalledYamls`?

Reply:
Ye, I will change it